src/hooks/useTier.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * This program has been developed by students from the bachelor Computer Science at Utrecht University within the Software Project course.
+ * © Copyright Utrecht University (Department of Information and Computing Sciences)
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { useEffect, useState } from 'react';
+import { BigNumber } from 'ethers';
+import { useAccount, useBlockNumber } from 'wagmi';
+
+import { useDiamondSDKContext } from '../context/DiamondGovernanceSDK';
+import { getErrorMessage } from '../lib/utils';
+
+export const useTier = () => {
```

Comment:
Can we have a comment here to explain what this hook is?

The other hooks in useFacetFetch.tsx I feel speak for themselves mostly, but this one's a bit less trivial (at least to me)